web/src/main/kotlin/no/nav/su/se/bakover/web/routes/søknadsbehandling/BehandlingJson.kt

```diff
@@ -22,4 +22,5 @@ internal data class BehandlingJson(
     val hendelser: List<HendelseJson>? = emptyList(),
     val stønadsperiode: StønadsperiodeJson?,
     val grunnlagsdataOgVilkårsvurderinger: GrunnlagsdataOgVilkårsvurderingerJson,
+    val erLukket: Boolean,
```

Comment:
Hva er mest hensiktsmessig/enklest for frontend av å utvide statusfeltet som allerede eksisterer kontra et til? Må man til legge til `&& !lukket` på alle eksisterende switches på status, eller finnes det noe union-magi som fikser dette på en enkel måte?